R/package.R

```diff
@@ -5,5 +5,6 @@
 #' @importFrom gDRutils fit_curves
 
 #' @keywords internal
+#' @return package help page
```

Comment:
@j-smola FYI (fix for the WARNING in the `BiocCheck()`)

Reply:
Thx 👍